bats_test/upload-file/single-bundled-files.bats

```diff
@@ -60,4 +59,22 @@ setup_file() {
 
     assert_line -n 0 --regexp '^(\w|-){43}$'
     assert_line -n 1 --regexp '^(\w|-){43}$'
-}
\ No newline at end of file
+}
+
+@test "Duplicate name uploads nothing with --skip" {
+    #We don't care about deleting this. Lives on Docker FS only
+    #We use quotes to escape possible spaces
+    run bash -c "touch '$PUB_FILE_NAME'"
+    #Again we use quotes to escape spaces
+    run -0 bash -c "yarn ardrive upload-file --dry-run --local-path ./'$PUB_FILE_NAME' -F $PARENT_FOLDER_ID -w $WALLET --skip | jq -r '.created | length'"
+
+    assert_output 0
+}
+
+@test "Duplicate name uploads a new file with --upsert" {
+    run bash -c "touch $PUB_FILE_NAME"
```

Comment:
nit
```suggestion
    run bash -c "touch '$PUB_FILE_NAME'"
```

Reply:
Beware the single quotes are working fine because the variables are addressed before passing the string to `bash`. If we weren't using bash here, then we must be using double-quotes.